packages/chaire-lib-frontend/src/components/captcha/SimpleCaptcha.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+// FIXME Using the NoReload captcha (which can pose problems to users) because the "Reload Captcha" text is hardcoded in english. Switch when https://github.com/masroorejaz/react-simple-captcha/pull/2 is accepted
```

Comment:
This have been accepted upstream already, should we update this PR (Like 4 years ago...)?

Reply:
> This have been accepted upstream already, should we update this PR (Like 4 years ago...)?

It's 1- 2 hours of testing and making sure it works, updating the code... if you insist, but we can open an issue of it, I feel my time is better spent somewhere else right now...

Reply:
I guess we could open an issue. Sad that this won't be touched for another 4 years...